workflow/workflow.py

```diff
@@ -46,21 +47,20 @@ def run(self):
         controller = self.cluster["head"]
         osds = self.cluster["osds"]
         pwd = os.path.abspath(os.path.join('..'))
+        run_deploy_args = ['redeploy']
         if len(self.cluster["rgw"]) and self.cluster["rgw_enable"]=="true":
-            with_rgw = True
-        else:
-            with_rgw = False
+            run_deploy_args.append('--with_rgw')
+        if not (self.cluster.has_key("parallel_deploy") and self.cluster["parallel_deploy"]=="false"):
+            run_deploy_args.append('--parallel_deploy')
```

Comment:
@liangzhaowang , oh, I see, but this is not desired way to do so, cetune will set default for all parameter, please ask shian for the way to do so